cohort_4/playbook/activities.md

```diff
@@ -11,6 +10,13 @@
     - [Feedback Forms](#feedback-forms)
 
 <!-- markdown-toc end -->
+# Activities
+
+
+## Hackers arrive
+
+Hacker attendance can be put into
+https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit.
```

Comment:
Perhaps make the wording a bit more direct?

Ex. "When a hacker arrives, mark their arrival in the attendance over at https://docs.google.com/spreadsheets/d/1Em5tUQdm4dXnthtWr4zY4KR9Q6JZMV_Hl2n6G9Ti9Bg/edit."